### DISCUSSION TITLE
VT Server Testing

### DISCUSSION THREAD

**Discussion by arcostasi** [Open-Agriculture/AgIsoStack-plus-plus#189] [General]
Is there any VT Server available for testing on Windows, Linux or Android? What are you using to test VT? Thanks

**Comment by ad3154** [comment]
We have some discussion on VT server here: https://github.com/ad3154/Isobus-plus-plus/issues/135 as it relates to this project.

The short version is that some of us are testing with physical VT terminals from various vendors. @GwnDaan Looked like maybe he as an Agco/Fendt NT01, and I have a Raven Viper4.

This project doesn't have an official VT server (yet). Graphics functionality is out of scope for this repo, but a VT server interface is planned at least to make integrating with a graphics framework as simple as possible. So the actual VT server components in this repo will be limited to a CAN interface and some object pool parsing. I would like to see a software VT based on this library get created as a separate repo some day.

Both myself and some other folks in the community are working on our own side-projects to make software VT servers, but it hasn't been a priority. Part of that is just because most of the community lacks GUI framework experience in general.
